src/simulation/environment/eclipse/_UnitTest/test_eclipse.py

```diff
@@ -408,6 +410,38 @@ def unitEclipseCustom(show_plots):
 
     return [testFailCount, ''.join(testMessages)]
 
+def test_shadow_vs_illumination_alias_and_deprecation_behavior():
+    """Check aliasing and deprecation behavior for EclipseMsgPayload."""
+    p = EclipseMsgPayload()
+
+    # Forward direction: setting illuminationFactor updates shadowFactor
+    p.illuminationFactor = 0.8
+    assert p.illuminationFactor == pytest.approx(0.8)
+    assert p.shadowFactor == pytest.approx(0.8)
```

Comment:
I'm not following what this test is doing?  line 420 is still throwing a depreciation warning.  When running all BSK unit tests, I should not be seeing a depreciation warning.  If I added `deprecated.filterwarnings("ignore", "p.shadowFactor")` I still saw the deprecation warning.